examples/Xunit.Microsoft.DependencyInjection.ExampleTests/appsettings.json

```diff
@@ -1,9 +1,5 @@
 ﻿{
   "Options": {
     "Rate": 10
-  },
```

Comment:
Hey @ChrisDoernen we did not need to remove anything or alter anything related to using secret values. Can you please undo this change? 
Let's put it in this way: I will only accept this PR if the changes are incremental. I do not want to get the already-implemented feature excluded or removed unless they are obsolete.